src/components/Link.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import classNames from "classnames";
+import PropTypes from "prop-types";
+
+import { ComponentRegister, LinkComponent } from "../ComponentRegister";
+
+import "lbh-frontend/lbh/core/_links.scss";
+
+/**
+ * A component for creating an anchor linking to other pages. You can customize the component used internally for
+ * integration with routers using {@link ComponentRegister.init}. By default, it uses an <a> tag.
+ *
+ * @param {string} [className]
+ * @param {string} href
+ * @param {ReactNode} children
```

Comment:
These are all child parameters of a required `props` object (that implements `LinkProps` when that exists).